src/Microsoft.VisualBasic.Forms/src/Microsoft.VisualBasic.Forms.vbproj

```
@@ -38,4 +39,23 @@
     </EmbeddedResource>
   </ItemGroup>
 
+  <!-- 
+    A temporary fix until the intellisense file will be added to the main docs bundle.
+    Refer to https://github.com/dotnet/winforms/issues/2395 for more infromation
+    -->
+  <Target Name="TempIntellisenseXmlFileSource" AfterTargets="CopyIntellisenseXmlsToTargetRefPath">
```

If a target using `AfterTargets` fails, it may not fail the build (silent failure). `BeforeTargets` or some other mechanism would be more reliable: https://github.com/microsoft/msbuild/issues/3345.